src/section-factoring-out-the-common-factor.ptx

```diff
@@ -299,45 +299,45 @@
 
     <subsection>
         <title>More Examples of Factoring out the Common Factor</title>
-
-        <p>The previous subsection did not have examples that cover every nuance with factoring out the greatest common factor. Here are a few more factoring examples that attempt to do so.</p>
-
+        <p></p>
         <example>
             <p>Factor <m>-35m^5+5m^4-10m^3</m>.</p>
 
-            <p>First, we identify the common factor. Instead of literally factoring each term as we did in earlier examples, it may help to recognize that <m>5</m> is the greatest common factor of the three coefficients (which were <m>-35</m>, <m>5</m>, and <m>-10</m>). And also to recognize that the largest power of <m>m</m> that divides all three terms is <m>m^3</m>. Therefore the greatest common factor is <m>5m^3</m>.</p>
+            <p>First, we identify the common factor. <m>5</m> is the greatest common factor of the three coefficients (which were <m>-35</m>, <m>5</m>, and <m>-10</m>) and also <m>m^3</m> is the largest expression that divides <m>m^5</m>, <m>m^4</m>, and <m>m^3</m>. Therefore the greatest common factor is <m>5m^3</m>.</p>
```

Comment:
We should never start a sentence with math. An overlapping issue is math should never follow punctuation.